database/buildSchema/45_activity_log.sql

```diff
@@ -296,14 +296,16 @@ BEGIN
                                 SELECT
                                     title, code, "index"
                                 FROM template_section
-                            WHERE
-                                code = ANY (ARRAY (
-                                        SELECT
-                                            assigned_sections
-                                        FROM review_assignment
-                                    WHERE
-                                        id = assignment_id))
-                                AND template_id = templ_id ORDER BY "index") t), 'level', level_num, 'isLastLevel', is_last_level, 'finalDecision', is_final_decision));
+                                WHERE
+                                    code = ANY (ARRAY (
+                                            SELECT
+                                                assigned_sections
+                                            FROM review_assignment
+                                            WHERE
+                                                id = assignment_id
+                                                and template_id = templ_id
+                                                and assigned_sections <> '{}' ))
+                                    AND template_id = 10 ORDER BY "index") t), 'level', level_num, 'isLastLevel', is_last_level, 'finalDecision', is_final_decision));
```

Comment:
I don't think you want to hard-code `template_id = 10` do you?

Reply:
Oops - Nice finding! I changed on the other one but forgot about this